include/gdf/cffi/types.h

```diff
@@ -91,6 +116,29 @@ typedef enum {
   N_GDF_AGG_OPS, /* additional aggregation ops should go BEFORE N_GDF_... */
 } gdf_agg_op;
 
+
+typedef enum {
+    GDF_ADD,
+    GDF_SUB,
+    GDF_MUL,
+    GDF_DIV,
+    GDF_TRUE_DIV,
+    GDF_FLOOR_DIV,
+    GDF_MOD,
+    GDF_POW,
+    //GDF_COMBINE,
+    //GDF_COMBINE_FIRST,
+    //GDF_ROUND,
```

Comment:
I don't think that it makes sense to have intermediate parts of an enum commented out.

Reply:
This can lead to API instabilities.

Reply:
I'm so sorry, I hadn't seen your message.
It is used only as an enum (not casting to integers).
It could be error-prone. In any case, it will be moved to the end.